apps/accounts/forms/auth.py

```diff
@@ -251,12 +276,20 @@ def clean(self):
                     pass
                 else:
                     self.inactive_user = True
-                    raise forms.ValidationError("That account is not active yet because you haven't confirmed your email. <a id=\"resend_confirmation\" href=\"javascript:void(0);\">Resend email confirmation &gt;</a>")
+                    raise forms.ValidationError(
+                        "That account is not active yet because you haven't confirmed your email. "
```

Comment:
please leave the string as 1 line and use `# noqa`

Reply:
Done